scripts/operations/node_management/ncn-ims-image-upload.sh

```diff
@@ -22,150 +22,160 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-# shellcheck disable=SC2086
 
-test -n "$DEBUG" && set -x
+test -n "${DEBUG}" && set -x
 set -eo pipefail
 
 unset CRAY_FORMAT
+UPDATE_CPC=true
+
+function err_exit
```

Comment:
I'm not familiar with all the nuance here, but I've seen that we use parens elsewhere in addition to the `function` keyword, so we should probably be consistent on this line and the function definitions below.

```suggestion
function err_exit()
```